tests/compile-fail/stacked_borrows/illegal_read2.rs

```diff
@@ -1,6 +1,7 @@
 // A callee may not read the destination of our `&mut` without
 // us noticing.
 
+#[rustfmt::skip]
```

Comment:
Why does this one need skipping?

Reply:
This is mentioned in the commit message &mdash; it's because rustfmt is buggy and produces a noisy error message whenever this file is formatted.

```console
error[internal]: left behind trailing whitespace
  --> tests/compile-fail/stacked_borrows/illegal_read2.rs:10:10:1
   |
10 |     
   | ^^^^
   |

warning: rustfmt has failed to format. See previous 1 errors.
```

I could alternatively try to tweak the contents of this file to work around the rustfmt bug.

Reply:
Oh, strange.
Ideally the attribute would come with a comment referencing the rustfmt bugreport. But just a quick `// work around rustfmt bug` would also do it.